generator/src/lib.rs

```diff
@@ -1331,58 +1340,32 @@ pub fn generate_extension_commands<'a>(
 
         generate_function_pointers(
             device_ident,
-            &extension_name,
+            Some((&vendor, &extension_ident)),
             &device_commands,
             &rename_commands,
             fn_cache,
             has_lifetimes,
         )
     });
 
-    quote! {
-        pub mod #extension_ident {
-            use super::*;
+    (
+        vendor,
+        quote! {
+            pub mod #extension_ident {
+                use crate::vk::extensions::*;
```

Comment:
Any reason to not use `super::super`?

Reply:
Multiple levels of relative references make for poor readability, IMO. I'm also not actually sure if that works.

Reply:
It does, and there's a comment now (on the generator side) explaining that this is strictly for including things imported at the top of the file (not outside of the `extenions.rs` file at least).

We could also define a `use super::*` on both `mod`s and get the same effect.

Reply:
It's generated code, and it works, so I'm happy regardless. No need for more churn.